stdlib/char.gr

```diff
@@ -191,3 +191,71 @@ provide let toString = (char: Char) => {
 
   result
 }
+
+/**
+ * Checks if the first value is less than the second value.
```

Comment:
Probably want to mention that it's checking if the first char is less than/greater than the other by Unicode scalar value.